api4/ldap_test.go

```diff
@@ -11,10 +11,7 @@ func TestTestLdap(t *testing.T) {
 	th := Setup().InitBasic()
 	defer th.TearDown()
 
-	_, resp := th.Client.TestLdap()
```

Comment:
Why are removing the permissions check here? should we simulate to have the license and check if a regular user can test ldap with the license or not?